web/views.py

```diff
@@ -54,7 +58,10 @@ def project_new(request):
 	return render(request, 'pages/project_edit.html', {'projectform': projectform})
 
 def project_edit(request, project_id):
-	project = get_object_or_404(Project, project_id=project_id)
+	if Project.objects.filter(pk=project_id).exists():
+		project = Project.objects.get(pk=project_id)
+	else:
+		project = None
```

Comment:
Instead of Project = None, just `return`:
`return render(request, 'pages/project_edit.html', {'invalid_project_id': True})`